src/endpoints/sub_accounts.js

```diff
@@ -0,0 +1,59 @@
+const roote = "/subAccounts";
+
+const subAccounts = {
+  createSubaccount: {
+    route: route,
+    method: "post",
+    BodyParams: {
+      required: [
+        "business_name",
+        "settlement_bank",
+        "account_number",
+        "percentage_charge",
+        "description",
+      ],
+      optional: [
+        "primary_contact_email",
+        "primary_contant_name",
+        "primary_contact_phone",
+        "metadata",
+      ],
+    },
+    listSubAccounts: {
+      route: route,
+      method: "put",
```

Comment:
This one is `get` and not `put`.
I case this is what @emmqnuelkipsqng meant method